images/influxdb/Dockerfile

```diff
@@ -13,7 +13,6 @@ RUN test -e /etc/nsswitch.conf || echo 'hosts: files dns' > /etc/nsswitch.conf
 
 RUN mkdir -p $INFLUXDB_HOME /data /logs
 
-ADD Dockerfile /
```

Comment:
I don't have a huge opinion on that but I thought it was a good idea to keep the source around.